### ISSUE TITLE
[Feature] Improved parsing logic

### ISSUE THREAD

**Issue by asagi4** [asagi4/comfyui-prompt-control#72]
PromptSchedule objects are currently pretty inflexible, and schedule filtering essentially re-parses everything

Maybe figure out better ways.

**Comment by asagi4** [comment]
It might make sense to separate the LoRA scheduling syntax away from the prompt scheduling syntax and use something closer to what the loractl extension for sd-webui does. This should make it more compact and less confusing.

**Comment by ansstuff** [comment]
Yes, the loractl syntax (specifying weights at keyframes) is the best, it's very intuitive and consize.
LoRA weight interpolation is one of the last remaining advantages of A1111 over ComfyUI.

Prompt interpolation doesn't seem to be as useful, though.

**Comment by diodiogod** [comment]
I also like loractl syntax, makes sense in my head while using lora inside the prompt scheduler syntax makes my head hurts. 😵
